### ISSUE TITLE
Error - destiantion address can't be resolved. Script 01_queryAddress.sh

### ISSUE THREAD

**Issue by HuguesD** [gitmachtl/scripts#43]
Hello,
I get the error : "destination address can't be resolved. Maybe filename wrong, or not a payment-address." running the command : 

"01_queryAddress.sh smallwallet1" on the airgap machine, on which the offlineTransfer.json has been copied from the online machine.

This offlineTransfer.json file has been created on the online machine with the following command : " 01_workOffline.sh add smallwallet1"

smallwallet1 was created succesfully and provisionned succesfully on the online machine with 605 ADA. The 01_queryAddress smallwallet1 works correctly on the online machine.

When I look into the offlineTransfer.json, I find lines mentionning "name : smallwallet1" and also its address.

So it seems that the 01_queryAddress.sh script does not find smallwallet1 for reason.

What did we do wrong or is it a bug ?

**Comment by gitmachtl** [comment]
HI,

1.) Is the offline machine working in offline mode? You have to set this in the setting (common.inc, 00_common.sh, etc.)
2.) Do you have the `smallwallet1.addr, smallwallet1.vkey, smallwallet1.skey` files on the offline machine? If not, you can attached them on the online machine via the `attach `command in 01_workOffline.sh and extract them on the offline machine via the `extract `command. Or you simply copy them over. 

There is no need to have the .vkey or .skey file on the online machine. On the online machine only the smallwallet1.addr file is needed.

**Comment by HuguesD** [comment]
Thanks. Ok I found a formatting error in the smallwallet1.addr file  : it was a strange character at the end. Probably due to the fact I transfered it from the online to the offline via WinSCP then USB key. So you're idea to use workoffline attach command will solve the formatting issue.
So we progress on this step but he next step has still an issue with the command
01_senLoveLaces.sh smallwallet1 ledgerowner.payment 6000000

This gives now the error : "smallwallet1.skey/hwsfile does not exist. Please ctreate it first with the script 03a or 02"
But smallwallet1 is a CLI wallet, not a hw wallet (only the ledger owner is a HW wallet)

Any idea what's wrong ?

**Comment by HuguesD** [comment]
Ok I found the bug : the three files addr, skey and vkey for smallwallet1 were in fact strangely enough with these names smallwallet1.addr (that is ok), smallwallet.skey and smallwallet.vkey (these names are wrong (missing "1")
Once I renamed themsmallwallet1.skey and smallwallet1.vkey, it works now !
I do not know how in the world these two files were produced with the wrong name ? especially the smallwallet1.addr was correct.
I do not remember having changed these names. I just copied them from the on line to the offline machine. Maybe the names were changed in the process, I do not know.
